observability/distr_tracing/distr-tracing-rn.adoc

```diff
@@ -12,9 +12,120 @@ You can use the {TempoName} xref:../otel/otel-forwarding-telemetry-data.adoc#ote
+==== Bug fixes
+
+This update introduces the following bug fixes:
+
+* Before this update, the {TempoShortName} failed on the {ibm-z-title} (`s390x`) architecture. With this update, the {TempoShortName} is available for the {ibm-z-title} (`s390x`) architecture. (link:https://issues.redhat.com/browse/TRACING-3545[TRACING-3545])
+* Before this update, the {TempoShortName} failed on clusters with non-private networks. With this update, you can deploy the {TempoShortName} on clusters with non-private networks. (link:https://issues.redhat.com/browse/TRACING-4507[TRACING-4507])
+* Before this update, the Jaeger UI might fail due to reaching a trace quantity limit, resulting in the *504 Gateway Timeout* error in the `tempo-query` logs. After this update, the issue is resolved by introducing two optional fields in the `tempostack` or `tempomonolithic` custom resource:
++
+** New `spec.timeout` field for configuring the timeout.
+** New `spec.template.queryFrontend.jaegerQuery.findTracesConcurrentRequests` field for improving the query performance of the Jaeger UI.
++
+[TIP]
+====
+One querier can handle up to 20 concurrent queries by default. Increasing the number of concurrent queries further is achieved by scaling up the querier instances.
+====
+
+[id="distr-tracing_3-4_jaeger-release-notes_{context}"]
+=== {JaegerName}
+
+The {JaegerName} 3.4 is provided through the link:https://catalog.redhat.com/software/containers/rhosdt/jaeger-operator-bundle/613b50f3a052bb39f2c5f31e/history[{JaegerOperator} Operator 1.62.0].
+
+[NOTE]
+====
+The {JaegerName} 3.4 is based on the open source link:https://www.jaegertracing.io/[Jaeger] release 1.62.0.
+====
+// 1.62.0 is fixed until Jaeger's removal
+
+[IMPORTANT]
+====
+Jaeger does not use FIPS validated cryptographic modules.
+====
+
+[id="distr-tracing_3-4_jaeger-release-notes_support-for-elasticsearch-operator_{context}"]
+==== Support for the {es-op}
+
+The {JaegerName} 3.4 is supported for use with the {es-op} 5.6, 5.7, and 5.8.
+
+[id="distr-tracing_3-4_jaeger-release-notes_deprecated-functionality_{context}"]
+==== Deprecated functionality
+
+In the {DTProductName} 3.4, Jaeger and support for Elasticsearch remain deprecated, and both are planned to be removed in a future release.
+Red{nbsp}Hat will provide support for these components and fixes for CVEs and bugs with critical and higher severity during the current release lifecycle, but these components will no longer receive feature enhancements.
+
+The {JaegerOperator} Operator link:https://access.redhat.com/solutions/7083722[will be removed] from the `redhat-operators` catalog in November 2025. You must xref:../otel/otel-migrating.adoc#dist-tracing-otel-migrating[migrate] to the xref:distr_tracing_tempo/distr-tracing-tempo-installing.adoc#dist-tracing-tempo-installing[{TempoOperator}] and the xref:../otel/otel-installing.adoc#install-otel[{OTELName}] for distributed tracing collection and storage.
```

Comment:
```suggestion
The {JaegerOperator} Operator link:https://access.redhat.com/solutions/7083722[will be removed] from the `redhat-operators` catalog in November 2025. You must xref:../../otel/otel-migrating.adoc#dist-tracing-otel-migrating[migrate] to the xref:../../distr_tracing_tempo/distr-tracing-tempo-installing.adoc#dist-tracing-tempo-installing[{TempoOperator}] and the xref:../../otel/otel-installing.adoc#install-otel[{OTELName}] for distributed tracing collection and storage.
```
Friendly reminder: `xrefs` need to walk back to the project root. So, the number of `../` in an `xref` is determined by how many directories an assembly is nested in. In this case, two directories.

Reply:
> Friendly reminder: `xrefs` need to walk back to the project root. So, the number of `../` in an `xref` is determined by how many directories an assembly is nested in. In this case, two directories.

@michaelryanpeter and @kalexand-rh, isn't relative path expected according to the repo's [`doc_guidelines.adoc`](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#internal-cross-references)?
![image](https://github.com/user-attachments/assets/f82323ff-0b39-41f8-ba78-a9aaab19cac7)

Reply:
> but you still have to back up to the root the project and then move forward

:slightly_smiling_face: I still don't understand why, @michaelryanpeter.
Are you aware of any toolchain reason for such a requirement?
And is this requirement written somewhere?

Reply:
Maybe some of the confusion is that I am use the term project root to describe your `openshift-docs` directory. 

Due to the way that our scripts convert xrefs into hard coded links for Pv1 and the structure of our repo, xrefs must back up to the project root and then use the complete path to the assembly your cross referencing. I do not know the specifics of why or how the script works, but I have seem broken builds and malformed links in Pantheon builds due to this issue. Many times.

Asciibinder does not have the same issue. It is much better at figuring out what your cross referencing and is more flexible in this regard.

TLDR: due to the way our repo is structured, you have to back up to your `openshift-docs` folder using the `../` prefix and then add the complete path from your `openshift-docs` folder to the assembly you are xrefing to.

The requirement is written in the OpenShift docs guidelines here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#internal-cross-references.

If you find the rule confusing or that it needs clarification, please open an issue or PR and we can discuss it at an upcoming OSDOCS and friends meeting. Or feel free to start a thread in #team-ocp-docs.

I hope this helps.